### ISSUE TITLE
Broadcast error

### ISSUE THREAD

**Issue by bidak** [vysheng/tg#628]
```
> broadcast contactName_contactLastName1 test
FAIL: 71: RPC_CALL_FAIL 400: CONTACTS_EMPTY
```

**Comment by bidak** [comment]
Please answer this question.

**Comment by kenniki** [comment]
Try using -I and specify the user using it's ID.

**Comment by bidak** [comment]
It does not work.

**Comment by bidak** [comment]
Is there any way?

**Comment by vysheng** [comment]
Did you use -W key?

**Comment by bidak** [comment]
yes.I used but does not work and shows contacts_empty.

**Comment by vysheng** [comment]
and can you send broadcast to them from phone?

**Comment by vysheng** [comment]
i'm not sure, but it is possible, that you can send broadcasts only to contacts

**Comment by bidak** [comment]
I have tested on phone so it didn't work! :|

**Comment by WuerfelDev** [comment]
Same problem here. Can it be because I renamed my contacts? Or because I need to use _--disable-python_ and _--disable-json_ in ./configure ?

**Comment by PouyaDarabi** [comment]
same issue here!

**Comment by mallakimahdi** [comment]
I have also have same problem

**Comment by nimix3** [comment]
Hello,
You can send message via Broadcast command only when the other one(user) added you as a contact...
You cannot send message to USERs that have not you on their contact list...
:D

**Comment by mallakimahdi** [comment]
@nimix3  i send broadcast to my contact but that error shown

**Comment by nimix3** [comment]
Yes, Because some or all of theme have not you (your number) on their phone !

**Comment by ronromer** [comment]
+1
